lang/en.yml

```diff
@@ -129,7 +171,7 @@ en:
     NEW: 'Add new record'
     NEXT: 'Go to next record'
     PREVIOUS: 'Go to previous record'
-    SAVEDUP: 'Saved successfully.'
+    SAVEDUP: 'Saved successfully'
```

Comment:
How come this is changing?

Reply:
.... It's a full stop. Does it really matter?
Either way, the PHP for that string in both CMS 5 and CMS 6 doesn't have that character so removing it seems correct.